# Position States
float64 xn
float64 yn
float64 xbd
float64 ybd
float64 xbdd
float64 ybdd
